sharding/notary/service.go

```diff
@@ -1,29 +1,23 @@
+// Package notary defines all relevant functionality for a Notary actor
+// within the minimal sharding protocol.
```

Comment:
Is it good to add minimal sharding protocol inside Package comments? That means we'll have to take it out after Ruby release